Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.h

```diff
@@ -37,9 +37,11 @@
 #include <wtf/Forward.h>
 #include <wtf/OptionSet.h>
 #include <wtf/RunLoop.h>
+#if USE(COORDINATED_GRAPHICS)
 #if PLATFORM(GTK)
 #include <WebCore/CoordinatedGraphicsLayer.h>
```

Comment:
I think we don't need to include CoordinatedGraphicsLayer.h in this header. We can add a forward declaration of CoordinatedGraphicsLayer.

Reply:
BTW, I don't like the idea using `USE(COORDINATED_GRAPHICS)` in the header.
If you have a time, please take a look. https://bugs.webkit.org/show_bug.cgi?id=232934#c5
But, it's not your problem. I don't object to land this patch if you fix the above the issue.

Reply:
I had a quick view on the topic. I was indeed also building with the option USE_OPENGL_OR_ES=OFF. (btw. I hope that this option won't be removed, because I need it.) I was able to build with my patch, but I'm open to remarks. I will check if I can forward declare that pointer. You can expect an update to this PR. But I won't refactor everything, I just want to keep my patch list small. (Similar as what the Yocto guys intended to do.)